packages/node-services/src/module.ts

```diff
@@ -84,7 +84,7 @@ export default class Module {
   }
 
   _compile(content: string, filename: string) {
-    var _self = this;
+    const _self = this;
```

Comment:
```suggestion
    var _self = this;
```

This is pretty important. This should not be a block variable, this is copied from the node code.